packages/api-tests/README.md

```diff
@@ -0,0 +1,8 @@
+# ContainerJS Testing
+This project contains the Spectron UI tests for the OpenFin and Electron APIs
+
+## Testing
+```
+npm i
+npm run test:ui
```

Comment:
This was because if it was npm test, lerna would run this when we run on travis (which would fail). We can change this if we manage to get OpenFin tests to work on linux.